utils/helpers/protocolHelper.ts

```diff
@@ -253,6 +290,55 @@ export class ProtocolHelper {
     );
   }
 
+  public async createDefaultRebalancingSetTokenV3Async(
```

Comment:
Does this helper not exist on `set-protocol-contracts`?

Reply:
Yeah it does. But, there was an issue importing this function from contracts. Similar issue to the one Brian ran into while importing this into setprotocol.js a few weeks ago